cmd/main.go

```diff
@@ -268,11 +313,7 @@ func ensureRequiredAPIGroupsAndResourcesExist(restConfig *rest.Config) {
 		},
 	}
 
-	discoveryClient, err := discovery.NewDiscoveryClientForConfig(restConfig)
-	if err != nil {
-		setupLog.Error(err, "failed to create discovery client")
-		os.Exit(1)
-	}
+	discoveryClient := discovery.NewDiscoveryClientForConfigOrDie(restConfig)
```

Comment:
huh? it doesn't return error now?

Reply:
It does, just *OrDie version is shorted (we did the same in the error handler)